### DISCUSSION TITLE
Spirit Level on the Hero12

### DISCUSSION THREAD

**Discussion by Squinkz** [gopro/labs#858] [General]
Any news on when the Hero 12 will get an update that fixes the spirit level? It broke after the last update and I've been sorely missing it ever since.

**Comment by dnewman-gpsw** [comment]
Works fine on the current firmware,

**Reply by Squinkz** [reply]
Ah you're right, somehow I completely forgot about the last firmware release.

My downloading zip file has a (1) next to it, which means I must have noticed the release when it came out and then just completely forgot about it! Thanks.
